master/buildbot/configurators/janitor.py

```diff
@@ -39,43 +40,67 @@ class LogChunksJanitor(BuildStep):
     name = 'LogChunksJanitor'
     renderables = ["logHorizon"]
 
-    def __init__(self, logHorizon):
+    def __init__(self, logHorizon=None, horizonPerBuilder=None):
```

Comment:
Buildbot wants to gradually move to `snake_case` naming style like the rest of Python ecosystem. Replace all `horizonPerBuilder` with `horizon_per_builder`.